src/main/java/frc/robot/commands/AimShoot.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * This command will use linear interpolation to adjust the
+ * shooter angle and shooter power based upon the distance from the target,
+ * which is determined using the robot pose estimator.
+ * 
+ * Parameters:
+ * 
+ * Swerve           (subsystem)
+ * ShooterPivot     (subsystem)
+ * Shooter          (subsystem)
+ * 
+ */
+
+package frc.robot.commands;
+
+import frc.robot.subsystems.Swerve;
+import frc.robot.subsystems.Eyes;
+import frc.robot.subsystems.Shooter;
+import frc.robot.subsystems.ShooterPivot;
+import edu.wpi.first.math.interpolation.InterpolatingDoubleTreeMap;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+
+
+public class AimShoot extends Command {
+
+    // required subystems
+    private Swerve swerve;
+    private Eyes eyes;
+    private ShooterPivot shooterPivot;  
+    private Shooter shooter;
+    private double distance;
+
+    // required WPILib class objects
+    private InterpolatingDoubleTreeMap shooterAngleInterpolation;
+    private InterpolatingDoubleTreeMap shooterSpeedInterpolation;
+
+    // local variables
+    private double shooterAngle;
+    private double shooterSpeed;
+
+    // constructor
+    public AimShoot(Swerve swerve, ShooterPivot shooterPivot, Shooter shooter) {
+
+        this.swerve = swerve;
+        this.shooterPivot = shooterPivot;
+        this.shooter = shooter;
+
+        addRequirements(swerve, shooterPivot, shooter);
+
+        // instantiate objects
+        shooterAngleInterpolation = new InterpolatingDoubleTreeMap();
+
+    }
+
+    @Override
+    public void execute() {
```

Comment:
The below code seems to be designed to only be run once, should it be in `execute()`?